<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input_Form</class>
 <widget class="QWidget" name="Input_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>device</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>34</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ip</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_ip"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>resolution</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_resolution">
      <item>
       <property name="text">
        <string>352*544</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>800*600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>720*576</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1280*720</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1920*1080</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>720*288</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>720*480</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>device type</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_devicetype">
      <item>
       <property name="text">
        <string>normaldev</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>multidev</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>virtualdev</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>238</y>
     <width>331</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_okay">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>okay</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
